packages/office-ui-fabric-react/src/components/Toggle/Toggle.base.tsx

```diff
@@ -101,14 +101,15 @@ export class ToggleBase extends React.Component<IToggleProps, IToggleState> impl
     // The following properties take priority for what Narrator should read:
     // 1. ariaLabel
     // 2. onAriaLabel (if checked) or offAriaLabel (if not checked)
-    // 3. label
-    // 4. onText (if checked) or offText (if not checked)
+    // 3. label AND stateText, if existent
+
     let labelledById: string | undefined = undefined;
     if (!ariaLabel && !badAriaLabel) {
       if (label) {
         labelledById = labelId;
-      } else if (stateText) {
-        labelledById = stateTextId;
+      }
+      if (stateText) {
```

Comment:
could you pls make the same fix in `useToggle` in `react-next` package? thanks!!